docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  sway:
+    build:
+      context: . 
+      dockerfile: deployment/Dockerfile
+    image: sway
```

Comment:
```suggestion
    image: sway

```

Reply:
apologies for some reason I am not getting these messages to my email -> I will figure it out -> this is fixed